openbas-api/src/test/java/io/openbas/rest/InjectApiTest.java

```diff
@@ -265,6 +267,71 @@ void deleteInjectsForScenarioTest() throws Exception {
 
   // -- EXERCISES --
 
+
+  @DisplayName("Add an inject for simulation")
+  @Test
+  @Order(9)
```

Comment:
this annotation is unnecessary unit testing function should be independant so they should be processed at any order